src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs

```diff
@@ -1789,7 +1789,6 @@ public static void DCS_KeyValuePair()
     }
 
     [Fact]
-    [ActiveIssue("ReflectionBasedDCS")]
     public static void DCS_ConcurrentDictionary()
```

Comment:
How does this change enable this test? ConcurrentDictionary doesn't derive from dictionary, it implements IDictionary<,> and IDictionary